example/passthrough/README.md

```diff
@@ -0,0 +1,45 @@
+# "Passthrough" setup for OpenTelemetry
+
+Some go programs may wish to propagate context without recording spans.  To do this in OpenTelemetry, simply install `TextMapPropagators`, but do not install a TracerProvider using the SDK.  This works because the default TracerProvider implementation returns a "Non-Recording" span that keeps the context of the caller, but does not record spans.
+
+For example, when you initialize your global settings, the following will propagate context without recording spans:
+
+```golang
+// Setup Propagators only
+otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{}))
+```
+
+But the following will propagate context _and_ record spans:
```

Comment:
```suggestion
But the following will propagate context _and_ create new, potentially recorded spans:
```

I think the important distinction here is that when a `TracerProvider` is configured new spans will be created that must be exported to ensure an unbroken trace.